elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/ElasticsearchQueries.java

```diff
@@ -28,7 +27,7 @@ private ElasticsearchQueries() {
 	 * documentation</a> for the complete query syntax.
 	 */
 	public static QueryDescriptor fromJson(String jsonQuery) {
-		return new ElasticsearchJsonQueryDescriptor( new JsonParser().parse( jsonQuery ).getAsJsonObject() );
+		return new ElasticsearchJsonQueryDescriptor( GsonHolder.PARSER.parse( jsonQuery ).getAsJsonObject() );
```

Comment:
@Sanne in fact, this is where I added a reference to the GsonHolder. So maybe, we should stick with the instantiation instead?